core/src/com/unciv/ui/screens/civilopediascreen/FormattedLine.kt

```diff
@@ -316,6 +296,28 @@ class FormattedLine (
         return table
     }
 
+    private fun renderExtraImage(labelWidth: Float): Table {
+        val table = Table(BaseScreen.skin)
+        fun getExtraImage(): Image {
+            if (ImageGetter.imageExists(extraImage))
+                return if (centered) ImageGetter.getDrawable(extraImage).cropToContent()
+                    else ImageGetter.getImage(extraImage)
+            val externalImage = ImageGetter.findExternalImage(extraImage)
+                ?: throw UncivShowableException("Extra image '[$extraImage]' not found")
+            return ImageGetter.getExternalImage(externalImage)
+        }
+        val image = getExtraImage()
+        // limit larger cordinate to a given max size
```

Comment:
For huge images, wouldn't we want to limit by labelWidth anyway? Sort of like .coerceAtMost?